<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChangeDCDialog</class>
 <widget class="QDialog" name="ChangeDCDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>509</width>
    <height>452</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Change Domain Controller</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QRadioButton" name="select_button">
     <property name="text">
      <string>Select:</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="select_listwidget"/>
   </item>
   <item>
    <widget class="QRadioButton" name="custom_button">
     <property name="text">
      <string>Custom:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="custom_edit">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="save_this_setting_check">
     <property name="text">
      <string>Save this setting</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="button_box">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>button_box</sender>
   <signal>accepted()</signal>
   <receiver>ChangeDCDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_box</sender>
   <signal>rejected()</signal>
   <receiver>ChangeDCDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>custom_button</sender>
   <signal>toggled(bool)</signal>
   <receiver>custom_edit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>254</x>
     <y>318</y>
    </hint>
    <hint type="destinationlabel">
     <x>254</x>
     <y>354</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>select_button</sender>
   <signal>toggled(bool)</signal>
   <receiver>select_listwidget</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>254</x>
     <y>24</y>
    </hint>
    <hint type="destinationlabel">
     <x>254</x>
     <y>171</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
